Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -195,6 +195,14 @@ def bump_formula_pr
             #{new_url}
         EOS
       end
+
+      if formula.resources.present? && old_url.include?("pythonhosted") && !args.force?
```

Comment:
I think this should apply for all formulae with `resources`, not just the `pythonhosted` ones.

For example, [`ansible`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/ansible.rb)'s url isn't a `pythonhosted` one but it does have pypi resources

Reply:
That's what I originally had but I didn't want to be too restrictive. I also considered iterating over the resources to see if there were any `pythonhosted` ones and only warn if a python resource was found. There are also non-python cases where resources need to be updaed, like https://github.com/Homebrew/homebrew-core/pull/57977, but I'm not sure if there's a great way to tell in general.

Reply:
My opinion is that this isn't too restrictive because there's the option of adding `--force`. The way I see it, the important part is that it raises the *possibility* that bumps are needed. There definitely are non-python formulae that need resource bumps but there also probably are formulae that don't ever need resource bumps.

I would argue that it never hurts to double-check (or, rather, suggest double-checking), and it's not too difficult to override with `--force`

Reply:
I think we should also check if the formula has any patches (as usually they are temporary fixes)

Reply:
For now I'm still just checking python formulae with resources, but it's also now checking formulae with patches as well.

Reply:
I'm not sure about the use of `--force` here. I think there should definitely be a warning, but maybe we don't want to stop the PR.

Additionally this should definitely work for all resources. I can't think of any that aren't bound to an upstream version.